guides/v2.3/install-gde/prereq/php-centos.md

```diff
@@ -0,0 +1 @@
+php-centos-ubuntu.md
```

Comment:
We try to avoid using symlinks as much as possible. Instead, I'd rather use a redirect in front-matter of the `php-centos-ubuntu.md`:
```
redirect_from:
  - /guides/v2.3/install-gde/prereq/php-centos.html
```

See https://github.com/jekyll/jekyll-redirect-from for more details about the feature.

Reply:
Thank you for the hint, will update the PR.

Reply:
The redirect was added in #3226